packages/shared/src/domain/enums/NetworkEnum.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { describe, it, expect } from 'vitest';
+import { NetworkEnum } from './NetworkEnum';
+
+describe('NetworkEnum', () => {
+  it('should have a MAINNET type with value "mainnet"', () => {
+    expect(NetworkEnum.MAINNET).toBe('mainnet');
+  });
+
+  it('should have a TESTNET type with value "testnet"', () => {
+    expect(NetworkEnum.TESTNET).toBe('testnet');
+  });
+});
```

Comment:
IMHO these tests add no value. What exactly is being tested here?

1- We should safely assume enums work as we expect, so we shouldn't test that
2- There's no logic being tested here. Just a definition

This is like testing a constant was defined with the value defined.

Reply:
Yes agreed. I don't think it's common to test enum definitions. I created this one because I saw another one pretty similar in the same folder (`TxTypesEnum`) and then I decided to follow the standard, although we can discuss it and I'm fine with getting rid with them.

Reply:
@rafappelt any thoughts on this?

Reply:
Yeah, I agree with @gndelia. These tests add no value. If I did something similar, I was wrong.

Reply:
All right. Removing them.